olive/passes/openvino/quantization.py

```diff
@@ -27,17 +27,14 @@ def _default_config() -> Dict[str, PassConfigParam]:
                 required=True,
                 description=(
                     "Specific config for openvino.tools.pot.IEEngine. 'engine_config' can be set"
-                    " by passing a dictonary, for example engine_config = {'device': 'CPU'}"
+                    " by passing a dictonary, for example engine_config: {'device': 'CPU'}"
                 ),
             ),
             "dataloader_func": PassConfigParam(
-                type_=Union[Callable, str],
+                type_=str,
```

Comment:
Is there a reason Callable is not supported anymore?

Reply:
Ah this was the test change I made before but forgot to change it back. Good catch!